cmake/FindBlosc.cmake

```diff
@@ -35,12 +35,24 @@
 # BLOSC_LIBRARYDIR      Blosc's library directory
 # BLOSC_LIBRARIES        all Blosc libraries
 
+SET ( BLOSC_SEARCH_PATHS
```

Comment:
I don't like all the duplication here, adding a new search path would require modifying all of these files, could this be moved somewhere where it can be shared?

Reply:
Indeed, I'll see if I can figure out a better way to handle this.